tests/src/main/java/com/hazelcast/simulator/tests/topic/ReliableITopicTest.java

```diff
@@ -0,0 +1,185 @@
+package com.hazelcast.simulator.tests.topic;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IAtomicLong;
+import com.hazelcast.core.ITopic;
+import com.hazelcast.core.Message;
+import com.hazelcast.core.MessageListener;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.simulator.test.TestContext;
+import com.hazelcast.simulator.test.TestRunner;
+import com.hazelcast.simulator.test.annotations.RunWithWorker;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.Teardown;
+import com.hazelcast.simulator.test.annotations.Verify;
+import com.hazelcast.simulator.utils.AssertTask;
+import com.hazelcast.simulator.worker.tasks.AbstractMonotonicWorker;
+import com.hazelcast.simulator.worker.tasks.IWorker;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+import static com.hazelcast.simulator.utils.CommonUtils.sleepRandomNanos;
+import static com.hazelcast.simulator.utils.TestUtils.assertTrueEventually;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests the {@link ITopic} with reliable configuration (backed by a ringbuffer). It creates a number of topics and a number of
+ * listeners per topic and each member publishes messages to every topic.
+ */
+public class ReliableITopicTest {
```

Comment:
Was just a start because I didn't knew yet if I would come up with some fancy verification stuff for the reliability. I'll merge the tests so we can just control the behavior over the `basename` property.